scripts/aws_ecs_benchmarks_cdn.sh

```diff
@@ -91,6 +104,8 @@ do
 done
 
 # shut down all related threads
+COMMAND_SHUTDOWN="for pid in $(ps -ef | grep "cdn-broker" | awk '{print $2}'); do kill -9 $pid; done"
```

Comment:
Here can we just do `killall -9 cdn-broker`? That way we don't have to get the PID explicitly